joshua/joshua_agent.py

```diff
@@ -460,6 +494,7 @@ def _run_ensemble(
         properties = joshua_model.get_ensemble_properties(ensemble)
         compressed = properties.get("compressed", False)
         command = properties.get("test_command", "./joshua_test")
+        done_command = properties.get("end_command", "./joshua_done")
```

Comment:
Should we pick use `done_command` as the key in the db?

Reply:
Sure, changed it